### ISSUE TITLE
Implement basic Osu! API

### ISSUE THREAD

**Issue by TheNiceGuy** [opendwellers/jujubot#2]
We could retrieve user/beatmap informations using the [API](https://github.com/ppy/osu-api). There is also the possibility to connect to Osu!'s IRC server located at [irc.osy.ppy](irc.ppy.sh) and send commands to [BanchoBot](https://osu.ppy.sh/wiki/BanchoBot).

**Comment by patoupatou** [comment]
Yes it should be fairly easy to have a osu subset of commands to the bot. Should it be @bot osu <command> <arguments> or @bot osu-<command> <arguments>?
